src/main/java/wooteco/subway/path/domain/Path.java

```diff
@@ -0,0 +1,46 @@
+package wooteco.subway.path.domain;
+
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+import wooteco.subway.line.domain.Section;
+import wooteco.subway.line.domain.Sections;
+import wooteco.subway.station.domain.Station;
+
+import java.util.List;
+
+public class Path {
+
+    private WeightedMultigraph<Station, DefaultWeightedEdge> graph;
+
+    public Path(Sections sections) {
+        this.graph = createGraph(sections);
+    }
+
+    public List<Station> findPath(Station source, Station target) {
+        DijkstraShortestPath dijkstraShortestPath = new DijkstraShortestPath(graph);
```

Comment:
```suggestion
        DijkstraShortestPath<Station, DefaultWeightedEdge> dijkstraShortestPath = new DijkstraShortestPath<>(graph);
```

제네릭 명시해주세요
다른 DijkstraShortestPath, WeightedMultigraph도 제네릭 빠진 부분 없는지 점검해보아요

Reply:
아하 제네릭을 명시해줄 수 있었군요..! 이번에 처음 라이브러리를 사용해봐서 몰랐습니다! 감사합니다!!!